pkg/process/metadata/parser/nodejs/nodejs_test.go

```diff
@@ -0,0 +1,45 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+package nodejs
+
+import (
+	"github.com/stretchr/testify/assert"
+	"path/filepath"
+	"testing"
```

Comment:
nit: sort the imports

```suggestion
	"path/filepath"
	"testing"
	
	"github.com/stretchr/testify/assert"
```